contracts/Bridge.sol

```diff
@@ -188,13 +188,14 @@ contract Bridge {
         }
         // Record vote
         _currentRelayerThresholdProposal._yesVotes[0] = msg.sender;
-        _currentRelayerThresholdProposal._hasVoted[msg.sender] = true;
+        _hasVotedOnRelayerThresholdProposal[proposalNonce][msg.sender] = true;
+        _relayerThresholdProposals[proposalNonce] = _currentRelayerThresholdProposal;
```

Comment:
im confused, this isn't a threshold change, why does it need to be here?

Reply:
So every time a Relayer calls `createRelayerThresholdProposal` we increment the `_relayerThresholdProposalNonce` and use it as the proposal's ID, much like with deposit proposals. This line:

```solidity
_relayerThresholdProposals[proposalNonce] = _currentRelayerThresholdProposal;
```

is just pairing the new proposal with it's ID, so that in the future, we have the option to look at all the threshold proposals and see which Relayer proposed it and which Relayers voted on it (or any other proposal data) - this part isn't a necessity right now, but 1. it was easy to add (and we'd get to hold onto the data for whatever future use case) and 2. It follows the pattern of the deposit proposals

If we wanted to go forward with this, I just realized I'd need to add it to `Relayer.sol` as well